doc/en/user/source/extensions/vectortiles/tutorial.rst

```diff
@@ -169,6 +169,115 @@ Create OpenLayers application
 
 These tiles are being rendered by the OpenLayers client.
 
+Create OpenLayers application - WMS Vector Tiles
+------------------------------------------------
+
+.. note::   Vector tiles requested with WMS allows to retrieve non-cached vector tiles (server side), setting the ``tiled=false`` parameter on the ``getMap`` request. This setting could be particularly useful when serving fast changing source data that should constantly be kept up-to-date for display.
```

Comment:
It should really be qualified by the size of the vector tile generated. One needs to pay attention to generate small ones (few attributes, few features) in order for them to be usable.

Also editing on vector tiles is problematic, geometry locations are quantized on a grid, you are never editing the original position, but a "close" one, how close depends on your current zoom level, just mind it will never be exactly the original one. Also, if dealing with lines/polygons crossing the tile borders, they are also clipped, reconstructing them on the client side from all the tiles containing them (good luck finding when to stop, think the coastline of Asia for example) is possible but causes further accuracy degradation.

Reply:
Included the qualification statement.